pallets/connectors/src/lib.rs

```diff
@@ -427,13 +449,83 @@ pub mod pallet {
 			)?;
 
 			Self::do_send_message(
-				who,
-				Message::Transfer {
+				who.clone(),
+				Message::TransferTrancheTokens {
 					pool_id,
 					tranche_id,
 					amount,
 					domain: domain_address.domain(),
-					address: domain_address.address(),
+					receiver: domain_address.address(),
+					sender: who
```

Comment:
I realize the order in this struct here doesn't matter technically, but I think for easy reading it would be nice to be consistent and have `sender` before `receiver` (and generally, just make sure the order of parameters matches that of the actual message parameter ordering)